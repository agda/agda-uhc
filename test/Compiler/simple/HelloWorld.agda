module HelloWorld where

open import Common.IO
open import Common.Unit

main : IO Unit
main = putStr "Hello World"
