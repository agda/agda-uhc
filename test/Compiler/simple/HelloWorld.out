EXECUTED_PROGRAM

ret > ExitSuccess
out > Hello World
